scripts/pylint_extensions.py

```diff
@@ -443,8 +430,6 @@ def visit_return(self, node):
             return
 
         func_node = node.frame()
-        if not isinstance(func_node, astroid.FunctionDef):
```

Comment:
Return statements can only be made inside a function else we will get an error from Python.